.circleci/config.yml

```diff
@@ -0,0 +1,217 @@
+version: 2
+workflows:
+  version: 2
+  tests:
+    jobs: &workflow_jobs
+      - node4:
+          filters:
+            tags:
+              only: /.*/
+      - node6:
+          filters:
+            tags:
+              only: /.*/
+      - node8:
+          filters:
+            tags:
+              only: /.*/
+      - node9:
+          filters:
+            tags:
+              only: /.*/
+      - node10:
+          filters:
+            tags:
+              only: /.*/
+      - lint:
+          requires:
+            - node4
+            - node6
+            - node8
+            - node9
+            - node10
+          filters:
+            tags:
+              only: /.*/
+      - docs:
+          requires:
+            - node4
+            - node6
+            - node8
+            - node9
+            - node10
+          filters:
+            tags:
+              only: /.*/
+      - system_tests:
```

Comment:
This repo does not have any system tests yet, so this part of the workflow can be removed (make sure to remove it from the `publish_npm` dependencies below as well).

Reply:
Instead, could we add a single system test that just passes? That way it's less work for the first person that wants to add one :)

Reply:
That's a good idea. If we can add a test that just connects and does nothing it would be easier to add real system tests later.